onegov/election_day/models/principal.py

```diff
@@ -52,3 +52,13 @@ def municipalities(self):
                 result[year] = {int(k): v for k, v in yaml.load(f).items()}
 
         return result
+
+    def year_available(self, year, map_required=True):
```

Comment:
Functions returning either True or False should be prefixed with `is_`. So `is_year_available` in this case.